### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by psychon**
'Nuff said. :-)

### CODE REVIEW

File: tests/run.sh [hunk 1]
```diff
@@ -91,31 +90,13 @@ AWESOME_CLIENT="$root_dir/utils/awesome-client"
...
-            echo "Log:"
-            cat "$awesome_log"
-            exit 1
-        fi
-        sleep 0.1
-    done
+    export DISPLAY="$D"
+    cd $root_dir/build
+    # Setup xrdb, for awesome's xresources backend / queries.
+    echo "Xft.dpi: 96" | DISPLAY="$D" xrdb
```
Comment (actionless) [inline tests/run.sh]: awesome anyway defaults to the same value so i don't think it's necessary

Comment (psychon) [inline tests/run.sh]: I *guess* that this will otherwise result in a warning that the grep at the end of this file would consider an error. Just a guess. Anyway, I didn't add this and shouldn't mess with it in this unrelated change.

A quick test results in:
    ...chon/projects/awesome/build/lib/beautiful/xresources.lua:67: W: Failed to get xrdb value 'Xft.dpi'.

The `grep` wouldn't trigger on this.

Comment (blueyed) [inline tests/run.sh]: > The grep wouldn't trigger on this.

Yes, but I found it cleaner to not trigger a warning there.
And it allows for adding other resources later etc.

**Comment by blueyed** [comment]
Thanks! :)
